app/models/comment.rb

```diff
@@ -0,0 +1,2 @@
+class Comment < ApplicationRecord
+end
```

Comment:
Why did you remove these lines? I think we need them for the polymorphic association. 
```
`belongs_to :commentable, polymorphic: true`
`validates :body, presence: true`
```
For the polymorphic association you should add them to make association works.

Also we may need to add `has_many :comments` to User Model so User can have many comments

Reply:
The polymorphic association  is so that the Comment model can be integrated into both the post and events as discussed. I'm not sure what happened to the comment model, I had to pull stuff from the repo before I could make my changes. I'll fix it.

Reply:
> The polymorphic association is so that the Comment model can be integrated into both the post and events as discussed. 

- I totally agree with it.  I was confused when noticed empty comment model.  Sure, after pulling last changes you will have opportunity to do that as we discussed with polymorphic model.  Let me know if you have any questions.